impl/client/src/main/javascript/web/pentaho/type/_type.js

```diff
@@ -901,9 +887,9 @@ define([
       /**
        * Gets the style classes of this and any base types.
        *
-       * Do **NOT** modify the returned array.
+       * Do **not** modify the returned array.
        *
-       * @type string[]
+       * @type {string[]}
```

Comment:
Not that it matters much, but did we settle on a preferred choice between `string[]` and `Array.<string>` ?